build/android-common.sh

```diff
@@ -63,8 +63,9 @@ get_library_name() {
         42) echo "libsndfile" ;;
         43) echo "leptonica" ;;
         44) echo "libsamplerate" ;;
-        45) echo "android-zlib" ;;
-        46) echo "android-media-codec" ;;
+        45) echo "cpu_features" ;;
```

Comment:
We used `-` (dash) instead of `_` (underscore) in all external library names. Can we continue following that approach?

Reply:
Yes, here there is less reason to stick to the original name. Especially if this library is enabled automatically.

Reply:
done